egs/librispeech/ASR/conformer_ctc/asr_datamodule.py

```diff
@@ -1 +1,594 @@
-../tdnn_lstm_ctc/asr_datamodule.py
\ No newline at end of file
+# Copyright      2021  Piotr Żelasko
```

Comment:
I think you changed this file from a soft link to a file.

Reply:
Yes. I changed it to a file.